docs/reference/index.md

```diff
@@ -1,40 +1,70 @@
-::::
+:::{note}
+For distributed tracing to work properly, your backend services must be configured to send telemetry to the {{stack}} as well.
+:::
 
+### Session review
 
-More detailed information on how the Agent works can be found in the [FAQ](/reference/faq.md#faq-how-does-it-work).
+The agent attaches [session](#session) info to each span and log generated from your application, allowing you to create queries where you can group all the telemetry that belongs to a session and form a session event timeline. This us useful to understand what are the most common actions performed by your users, as well as tracing their steps towards errors they may encounter.
 
+For example, let's say you have a screen "A" in your app that can be opened from other screens, such as "B". If you create a log event for when the user clicks on a button on screen "B" that takes them to screen "A", along with a log when screen "A" opens (or maybe a span instead, in case you'd like to measure how long it takes for screen "A" to fully load), both items will contain a `session.id` attribute with the same value per session. This enables you to create {{es}} queries, say in {{kib}}'s [Discover tool](https://www.elastic.co/guide/en/kibana/current/discover.html) for example, to list all events during that session and better understand your user's journey within your application.
 
-## Additional components [additional-components]
+### More
 
-APM Agents work in conjunction with the [APM Server](docs-content://solutions/observability/apps/application-performance-monitoring-apm.md), [Elasticsearch](docs-content://get-started/index.md), and [Kibana](docs-content://get-started/the-stack.md). The [APM Guide](docs-content://solutions/observability/apps/application-performance-monitoring-apm.md) provides details on how these components work together, and provides a matrix outlining [Agent and Server compatibility](docs-content://solutions/observability/apps/apm-agent-compatibility.md).
+The examples above highlight a couple of use cases that you can achieve with the agent and the {{stack}}. However, since the agent not only configures the [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/java/) but also provides [direct access](manual-instrumentation.md) to its features, you can generate your own data in ways that best suit your needs and take advantage of {{stack}}'s tools, such as [creating alerts](https://www.elastic.co/guide/en/kibana/current/alerting-getting-started.html) for when something interesting for you happens (maybe when an error is recorded), as well as [custom dashboards](https://www.elastic.co/guide/en/kibana/current/dashboard.html) to display your data the way you need to see it, and [much more](https://www.elastic.co/kibana/features).
```

Comment:
I think there are opportunities to tighten this up and make it more digestible. 

```suggestion
Since the agent not only configures the [OpenTelemetry SDK](https://opentelemetry.io/docs/languages/java/) but also provides [direct access](manual-instrumentation.md) to its features, you can generate your own data in ways that best suit your needs and take advantage of {{stack}}'s tools, such as:

 * [Creating alerts](https://www.elastic.co/guide/en/kibana/current/alerting-getting-started.html) when something interesting happens (for example, when an error is recorded)
 * [Building custom dashboards](https://www.elastic.co/guide/en/kibana/current/dashboard.html) to display your data the way you need to see it
 * [Much more](https://www.elastic.co/kibana/features)
```